tests/filecheck/dialects/arith/canonicalize.mlir

```diff
@@ -15,3 +15,22 @@
 // CHECK-NEXT:   %addf = arith.addf %lhsf32, %rhsf32 : f32
 // CHECK-NEXT:   %addf_vector = arith.addf %lhsvec, %rhsvec : vector<4xf32>
 // CHECK-NEXT:   "test.op"(%addf, %addf_vector) : (f32, vector<4xf32>) -> ()
+
```

Comment:
Nit: I believe we prefer to put filecheck tests above the test now? If so this file is small enough to fix imo